### PR TITLE
Fixed outdated/invalid classpath entries in MANIFESTs

### PR DESCRIPTION
**Pull request by Flugtiger**

### CODE REVIEW

File: engine/org.eclipse.birt.report.engine.emitter.config/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -3,8 +3,7 @@ Bundle-ManifestVersion: 2
 Bundle-Name: BIRT Emitter Configuration Plug-in
 Bundle-SymbolicName: org.eclipse.birt.report.engine.emitter.config;singleton:=true
 Bundle-Version: 4.8.0.qualifier
-Bundle-ClassPath: library.jar,
```
Comment (yulinwang) [inline engine/org.eclipse.birt.report.engine.emitter.config/META-INF/MANIFEST.MF]: I do find library.jar in this plugin, so cannot simply remove this from classpath.

**Comment by yulinwang** [comment]
I find these internal jars are used by fragments instead of host plugin itself.
I need further verification to ensure it won't impact class loading.

**Comment by Flugtiger** [comment]
As far as I could tell, none of those jars I removed are even existent anymore.

**Comment by yulinwang** [comment]
We find a similar regression which is related to change https://github.com/eclipse/birt/commit/66d91e3f64fddccb7361635939a293de4e19f4db.
The library.jar in classpath cannot be removed now, because it's used to be overridden by fragment.
I am still thinking how to remove this unnecessary reference in host plugin while keep fragment working.

**Comment by Flugtiger** [comment]
ok, i'll close this pull request until there is a solution

**Pull request closed without merging**